config/params.ini

```diff
@@ -97,6 +97,7 @@ always_repair=0
 id_items=0
 open_chests=1
 fill_shared_stash_first=0
+have_teleport_charges=0
```

Comment:
are you able to let the bot figure out whether teleport is a charged skill or not on 1st run? might be able to avoid having to add a param if so.

Reply:
This actually a good idea, less settings less questions why it doesn't work :)

Reply:
Done!